### PR TITLE
fix reference issue where lastupdatedby was null

### PR DESCRIPTION
**Pull request by mrjsawdk**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [x] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [x] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [ ] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Presentation.Web/Controllers/API/V1/DataProcessingRegistrationController.cs [hunk 1]
```diff
@@ -758,8 +758,11 @@ private List<DataProcessingRegistrationDTO> ToDTOs(IQueryable<DataProcessingRegi
             return value
                 .Include(dataProcessingRegistration => dataProcessingRegistration.Rights)
                 .Include(dataProcessingRegistration => dataProcessingRegistration.ExternalReferences)
+                .Include(dataProcessingRegistration => dataProcessingRegistration.ExternalReferences.Select(x=>x.LastChangedByUser))
+                .Include(dataProcessingRegistration => dataProcessingRegistration.ExternalReferences.Select(x=>x.ObjectOwner))
                 .Include(dataProcessingRegistration => dataProcessingRegistration.Reference)
                 .Include(dataProcessingRegistration => dataProcessingRegistration.Reference.ObjectOwner)
+                .Include(dataProcessingRegistration => dataProcessingRegistration.Reference.LastChangedByUser)
```
Comment (JacobMalling) [inline Presentation.Web/Controllers/API/V1/DataProcessingRegistrationController.cs:765]: Bruges ObjectOwner og LastChangedByUser i DPR overview? Den del her er jo kun til MasterReference

File: Presentation.Web/app/components/it-contract/tabs/it-contract-tab-reference.ts [hunk 2]
```diff
@@ -81,6 +81,7 @@
                     }, {
                         field: "lastChangedByUser.fullName",
                         title: "Senest ændret af",
+                        template: dateItem => dateItem.lastChangedByUser?.fullName || dateItem.objectOwner?.fullName,
```
Comment (JacobMalling) [inline Presentation.Web/app/components/it-contract/tabs/it-contract-tab-reference.ts:84]: Vi bør ikke have det sidste fallback med ""? I tilfælde noget der er gået helt galt

File: Presentation.Web/app/components/it-project/tabs/it-project-tab-reference.ts [hunk 3]
```diff
@@ -81,6 +81,7 @@
                     }, {
                         field: "lastChangedByUser.fullName",
                         title: "Senest ændret af",
+                        template: dateItem => dateItem.lastChangedByUser?.fullName || dateItem.objectOwner?.fullName,
```
Comment (JacobMalling) [inline Presentation.Web/app/components/it-project/tabs/it-project-tab-reference.ts:84]: Sidste fallback?

File: Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-reference.ts [hunk 4]
```diff
@@ -81,6 +81,7 @@
                 }, {
                     field: "lastChangedByUser.fullName",
                     title: "Senest ændret af",
+                    template: dateItem => dateItem.lastChangedByUser?.fullName || dateItem.objectOwner?.fullName,
```
Comment (JacobMalling) [inline Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-reference.ts:84]: Sidste fallback?

**Review by JacobMalling — APPROVED** [review]

File: Presentation.Web/Controllers/API/V1/DataProcessingRegistrationController.cs (hunk 1 above)
Comment (mrjsawdk) [reply Presentation.Web/Controllers/API/V1/DataProcessingRegistrationController.cs:765]: mja det er samme "mapper" i controlleren og eftersom det er samme identity vil den kun hente objektet 1 gang

**Pull request merged**